widget/embedded/src/constants/quote.ts

```diff
@@ -21,3 +22,19 @@ export type FeesGroup = {
   payable: { [key in NameOfFees]?: SwapFee[] };
   nonePayable: { [key in NameOfFees]?: SwapFee[] };
 };
+
+export const ROUTE_SORTING_STRATEGY: { [key in SortStrategy]: PreferenceType } =
+  {
+    'Fastest Transfer': 'SPEED',
+    'Maximum Return': 'NET_OUTPUT',
+    'Lowest Fee': 'FEE',
+    'Maximum Output': 'PRICE',
+  };
+
```

Comment:
I believe it would be preferable to swap the keys and values, then pass the values to the translate function: 
SPEED: t('Fastest Transfer')

Reply:
fixed